bin/package_build.py

```diff
@@ -134,7 +134,7 @@ def fedora():
 				req = requests.get(link)
 				data = req.text
 				if req.status_code == 404:
-					raise Exception(f"404 Directory {dirs[each]} not found")
```

Comment:
@Princee215 lets keep a verbose message to let users atleast know the issue they are facing

Reply:
@Princee215 resolved